cmd/storer2elastic/main.go

```diff
@@ -0,0 +1,283 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+
+	"github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/cmd/storer2elastic/config"
+	"github.com/ElrondNetwork/elrond-go/cmd/storer2elastic/databasereader"
+	"github.com/ElrondNetwork/elrond-go/cmd/storer2elastic/dataprocessor"
+	"github.com/ElrondNetwork/elrond-go/cmd/storer2elastic/elastic"
+	nodeConfigPackage "github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/core"
+	stateFactory "github.com/ElrondNetwork/elrond-go/data/state/factory"
+	"github.com/ElrondNetwork/elrond-go/data/typeConverters"
+	"github.com/ElrondNetwork/elrond-go/data/typeConverters/uint64ByteSlice"
+	"github.com/ElrondNetwork/elrond-go/hashing"
+	hasherFactory "github.com/ElrondNetwork/elrond-go/hashing/factory"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	marshalFactory "github.com/ElrondNetwork/elrond-go/marshal/factory"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/ElrondNetwork/elrond-go/storage/factory"
+	"github.com/ElrondNetwork/elrond-go/storage/storageUnit"
+	"github.com/urfave/cli"
+)
+
+type flags struct {
+	dbPathWithChainID  string
+	configFilePath     string
+	nodeConfigFilePath string
+	nodesSetupFilePath string
+	numShards          int
+	timeout            int
+}
+
+var (
+	nodeHelpTemplate = `NAME:
+   {{.Name}} - {{.Usage}}
+USAGE:
+   {{.HelpName}} {{if .VisibleFlags}}[global options]{{end}}
+   {{if len .Authors}}
+AUTHOR:
+   {{range .Authors}}{{ . }}{{end}}
+   {{end}}{{if .Commands}}
+GLOBAL OPTIONS:
+   {{range .VisibleFlags}}{{.}}
+   {{end}}
+VERSION:
+   {{.Version}}
+   {{end}}
+`
+	// configFilePathFlag defines a flag which holds the configuration file path
+	configFilePathFlag = cli.StringFlag{
+		Name:        "config",
+		Usage:       "This string file specifies the `filepath` for the toml configuration file",
+		Value:       "./config.toml",
+		Destination: &flagsValues.configFilePath,
+	}
+
+	// address defines a flag for setting the address and port on which the node will listen for connections
+	dbPathWithChainIDFlag = cli.StringFlag{
+		Name:        "db-path",
+		Usage:       "This string flag specifies the path for the database directory, the chain ID directory",
+		Value:       "db/chainID",
+		Destination: &flagsValues.dbPathWithChainID,
+	}
+
+	// nodeConfigFilePathFlag defines a flag which holds the configuration file path
+	nodeConfigFilePathFlag = cli.StringFlag{
+		Name:        "node-config",
+		Usage:       "This string file specifies the `filepath` for the node's toml configuration file",
+		Value:       "../node/config/config.toml",
```

Comment:
make it relative to this binary. I do not think this binary will run near a node as in case of emergency re-indexing we will run this separately

Reply:
based on the path of this binary, it can remain like this. This way, the tool will run with the exact configuration as the node. The flag value can be overwritten, anyway